aries-site/src/data/structures/components.js

```diff
@@ -751,6 +751,36 @@ export const components = [
     relatedContent: ['DataTable', 'Lists', 'Card'],
     tags: ['paginated results', 'paginated data'],
   },
+  {
+    name: 'Tag',
+    available: true,
+    category: 'Text',
+    description: `Tags are metadata attributes to facilitate identification, 
+    organization, searching and filtering resources.`,
+    preview: {
+      image: {
+        src: {
+          light: '/componentImages/component-preview-pagination.svg',
+          dark: '/componentImages/component-preview-pagination-invert.svg',
+        },
+        alt: 'HPE Design System Pagination',
```

Comment:
Are these tbd? The 'pagination' looks like copy paste...

Reply:
Yes these are TBD, thanks -- I'll fill them in.